app/controllers/levels_controller.rb

```diff
@@ -7,4 +7,23 @@ def show
     @level = Level.find(params[:id])
     @count_of_monsters_level = @level.monsters.count
   end
+
+  def new
+
+  end
+
+  def create
+    create_level = Level.create(name: params[:name], boss: params[:boss], difficulty: params[:difficulty] )
+    redirect_to "/levels"
+  end
+
+  def edit
+    @level = Level.find(params[:level_id])
+  end
+
+  def update
+    @level = Level.find(params[:level_id])
+    Level.update(name: params[:name], boss: params[:boss], difficulty: params[:difficulty] )
```

Comment:
do you need to add an id here to update the correct record?

Level.update(**@level.id,** name: params[:name], boss: params[:boss], difficulty: params[:difficulty] )